extensions/wikia/Rail/RailController.class.php

```diff
@@ -13,6 +13,7 @@ public function executeIndex($params) {
 
 		$railModules = isset($params['railModuleList']) ? $params['railModuleList'] : [];
 
+		$this->isEditPage = isset($params['isEditPage']) ? $params['isEditPage'] : false;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)